resources/views/partials/front-page-hero.blade.php

```diff
@@ -0,0 +1,10 @@
+<div class="row">
+    <div class="col-md-6">
+        <h2>{!! htmlspecialchars($get_hero_page->post_title) !!}</h2>
```

Comment:
please refer to documentation on the correct use of `!!` https://laravel.com/docs/master/blade#displaying-data Title is 'Displaying Unescaped Data'

Reply:
cool...any related thoughts on leaving it un-escaped in the next two elements?

Reply:
Yes, looks like not a good idea... here's the results: ![results](https://user-images.githubusercontent.com/17072191/52020118-f8dd4d00-24a4-11e9-93b2-b3e2864ea687.png)

Although can do it for the get_permalink... will commit now